packages/ref-core/src/cmip_ref_core/pycmec/output.py

```diff
@@ -0,0 +1,55 @@
+from enum import StrEnum
+from typing import Optional
+
+from pydantic import (
+    BaseModel,
+)
+from pydantic.json_schema import GenerateJsonSchema, JsonSchemaValue
+
+
+class OutputCV(StrEnum):
+    """CMEC output bundle controlled vocabulary"""
+
+    INDEX = "index"
+    PROVENANCE = "provenance"
+    DATA = "data"
+    PLOTS = "plots"
+    HTML = "html"
+    METRICS = "metrics"
+    FILENAME = "filename"
+    LONG_NAME = "long_name"
+    DESCRIPTION = "description"
+
+
+class OutputIndex(BaseModel):
+    """CMEC output bundle index object"""
+
+    index: str = "index.html"
+
+
+class OutputProvenance(BaseModel):
+    """CMEC output bundle provenance object"""
+
+    environment: dict  # Key/value pairs listing all relevant diagnostic and framework environment variables.
```

Comment:
The following will be more consistent with the rest of the project

```suggestion
    environment: dict[str, str]  
    """Key/value pairs listing all relevant diagnostic and framework environment variables."""
```

`dict` needs hints for the types of the keys and values. See another comment below